### PR TITLE
New Recipe /// Ration Coffee

### PR DESCRIPTION
**Pull request by Caelphon**
For some time now I've been wanting to add this, mostly to relate to some of the backgrounds from Adhomai (military) that would not get "good" coffee but instead, shitty ration coffee. 

This adds a new recipe (Ration Coffee) for those that want to replicate their experiences from wherever they've come from that didn't have good coffee! 

This is my first PR, and my first attempt at actually working with Github. Forgive me if I butchered this completely.

### CODE REVIEW

**Comment by Caelphon** [comment]
!review

**Comment by Caelphon** [comment]
It worked fine on a test server, by the way. So I think it should be okay.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -1762,6 +1762,17 @@
 	glass_desc = "A strong coffee made by passing nearly boiling water through coffee seeds at high pressure."
 	glass_center_of_mass = list("x"=15, "y"=9)
 
+/singleton/reagent/drink/coffee/ration_coffee
```
Comment (SleepyGemmy) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:1765]:
```suggestion
/singleton/reagent/drink/coffee/ration
```

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -2369,6 +2369,13 @@
 	required_reagents = list(/singleton/reagent/alcohol/rum = 1, /singleton/reagent/water = 1)
 	result_amount = 2
 
+/datum/chemical_reaction/drink/ration_coffee
+    name = "Ration Coffee"
+    id = "ration_coffee"
+    result = /singleton/reagent/drink/coffee/ration_coffee
```
Comment (SleepyGemmy) [inline code/modules/reagents/Chemistry-Recipes.dm:2375]:
```suggestion
    result = /singleton/reagent/drink/coffee/ration
```

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**